### PR TITLE
Make transaction_ignore_urls central config compatible

### PR DESCRIPTION
**Pull request by gregkalapos**
Solves https://github.com/elastic/apm-agent-dotnet/issues/1086

This is based on https://github.com/elastic/apm-agent-dotnet/pull/1082 - Same approach.

We could review https://github.com/elastic/apm-agent-dotnet/pull/1082 first and merge and then review this.

### CODE REVIEW

File: src/Elastic.Apm/BackendComm/CentralConfig/CentralConfigResponseParser.cs [hunk 1]
```diff
@@ -136,30 +136,37 @@ internal class CentralConfigPayload
...
+				LogLevelKey,
+				SpanFramesMinDurationKey,
+				StackTraceLimitKey
 			};
 
 			private readonly IDictionary<string, string> _keyValues;
 
 			public CentralConfigPayload(IDictionary<string, string> keyValues) => _keyValues = keyValues;
 
+			public string this[string key] => _keyValues.ContainsKey(key) ? _keyValues[key] : null;
```
Comment (russcam) [inline src/Elastic.Apm/BackendComm/CentralConfig/CentralConfigResponseParser.cs:166]: `TryGetValue()` and return the `out` parameter?

**Review by russcam — APPROVED** [review]
LGTM 👍

File: src/Elastic.Apm/BackendComm/CentralConfig/CentralConfigResponseParser.cs (hunk 1 above)
Comment (gregkalapos) [reply src/Elastic.Apm/BackendComm/CentralConfig/CentralConfigResponseParser.cs]: Ok, changed it. I liked the 1-liner, but reading from the `IDictionary` only once is definitely better.

**Comment by gregkalapos** [comment]
Integration test failed, but based on logs it's unrelated to this PR and it also failed on master - I suspect a general integration test problem. On the other hand the .NET pipeline is green.

Merging this in.

**Pull request merged**